lib/applications.js

```diff
@@ -596,7 +595,6 @@ exports.postApplication = async (req, res) => {
 
         // Sending the mail to a user.
         await mailer.sendMail({
-            from: 'oms-mailer@aegee.org',
```

Comment:
where did the `from` header go?

Reply:
the default one is used instead, we are using mailer@aegee.eu now

Reply:
ok, make sure it is "hidden" from source code and passed as variable (also to not hardcode it but mainly to not get spam by address crawlers)

Reply:
> and passed as variable

it's not passed, if we don't pass this field to mailer it'll use the default one, the mailer will take the one in its env. I don't have it stored anywhere in oms-statutory/events/discouns, only in mailer as default env variable.
and we cannot send emails from other accounts now, only from mailer@aegee.eu, that's GSuite limitations afaik